Source/inv.cpp

```diff
@@ -2065,7 +2065,11 @@ BOOL UseStaff()
 {
 	if (pcurs == CURSOR_HAND) {
 		if (plr[myplr].InvBody[INVLOC_HAND_LEFT]._itype != ITYPE_NONE
-		    && plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF
+#ifdef HELLFIRE
+		   (plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF || plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_UNIQUE)

```

Comment:
Hellfire doesn't compile since you missed `&&` from the line

```suggestion
		   && (plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF || plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_UNIQUE)
```

Reply:
I am very sorry. I will fix this immediately!

Reply:
added your indentation.